### ISSUE TITLE
Expand on Cypress Implementation

### ISSUE THREAD

**Issue by Linda-Vu** [Gary-Community-Ventures/benefits-api#611]
We want to implement end-to-end (E2E) tests using Cypress, including happy paths and edge cases. This will make it easier to catch potential regressions and bugs early in the development process and improve release confidence. The [manual QA regression testing plan](https://docs.google.com/spreadsheets/d/1CDydtWmvRu7qlwyzJLiHHkC4-_103upOw7z_Avbj28E/edit?gid=995642823#gid=995642823) can be used as a starting point. 

We have an existing implementation from 2023, but may need to adjust it to capture more scenarios https://github.com/Gary-Community-Ventures/benefits-calculator/blob/main/cypress/e2e/user_flow_input_test.cy.js

Requirements:
- Test coverage should include core features, data validation, input field constraints, and error messages
- Tests should not have text values hard-coded such that they fail when language is changed

**Comment by Linda-Vu** [comment]
@sydney-devine & @CalebPena wanted to make y'all aware that we are interested in this! @kbrey is looking into the technical scope right now before we actually assign it to anyone. is that cool? let us know if there's anything we should be aware of with regards to the implementation y'all did in 2023!

**Comment by sydney-devine** [comment]
> @sydney-devine & @CalebPena wanted to make y'all aware that we are interested in this! @kbrey is looking into the technical scope right now before we actually assign it to anyone. is that cool? let us know if there's anything we should be aware of with regards to the implementation y'all did in 2023!

Hey let's chat about it in our meeting next week? I'm not 100% sure what this is related to?

**Comment by Linda-Vu** [comment]
@sydney-devine sure! it's related to us wanting to standardize our QA process for new features that we're adding. especially for testing new programs and the accuracy of their numbers. we want to leverage cypress to do this. since we found that there was an existing implementation of cypress from Ivonne (linked in the ticket description), we figured that we could expand on that, and wanted a gut check from you and @CalebPena to see if that was cool!
